test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -395,6 +398,62 @@ checkpointerTest name initdata =
 
           _cpDiscard _cpeCheckpointer
 
+          next "2nd mini-regression test for debugging updates (part 1) empty block"
+
+          hash14 <- BlockHash <$> merkleLogHash "0000000000000000000000000000004b"
+
+          _blockenv04 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 4, hash13))
+
+          _cpSave _cpeCheckpointer hash14
+
+          next "2nd mini-regression test for debugging updates (part 2) create module & table"
+
+          hash15 <- BlockHash <$> merkleLogHash "0000000000000000000000000000005c"
+
+          blockEnv06 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 5, hash14))
+
+          void $ runExec blockEnv06 (Just $ ksData "6") $ defModule "6"
+
+          runExec blockEnv06 Nothing "(m6.readTbl)" >>= \EvalResult{..} -> Right _erOutput @?= traverse toPactValue [tIntList [1]]
+
+          _cpSave _cpeCheckpointer hash15
+
+          next "2nd mini-regression test for debugging updates (part 3) step 1 of insert value then update twice"
+
+          hash06 <- BlockHash <$> merkleLogHash "0000000000000000000000000000006a"
+
+          blockEnv07 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 6, hash15))
+
+          void $ runExec blockEnv07 Nothing "(m6.insertTbl 'b 2)"
+
+          _cpSave _cpeCheckpointer hash06
+
+          next "2nd mini-regression test for debugging updates (part 4) step 2 of insert value then update twice"
+
+          hash07 <- BlockHash <$> merkleLogHash "0000000000000000000000000000007a"
+
+          blockEnv08 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 7, hash06))
+
+
+          void $ runExec blockEnv08 Nothing "(m6.weirdUpdateTbl 'b 4)"
+
+          _cpSave _cpeCheckpointer hash07
+
+          next "2nd mini-regression test for debugging updates (part 5) step 3 of insert value then update twice"
+
+          hash08 <- BlockHash <$> merkleLogHash "0000000000000000000000000000008a"
+
+          blockEnv09 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 8, hash07))
+
+          -- FOR DEBUGGING/INSPECTING VALUES AT SPECIFIC KEYS
+          -- void $ runExec blockEnv09 Nothing "(let ((written (at 'col (read m6.tbl 'a [\"col\"])))) (enforce (= written 1) \"key a\"))"
+          -- void $ runExec blockEnv09 Nothing "(let ((written (at 'col (read m6.tbl 'b [\"col\"])))) (enforce (= written 4) \"key b\"))"
+          -- FOR DEBUGGING/INSPECTING VALUES AT SPECIFIC KEYS
+
+          runExec blockEnv09 Nothing "(m6.readTbl)" >>= \EvalResult{..} -> Right _erOutput @?= traverse toPactValue [tIntList [1,4]]
```

Comment:
It probably won't be obvious to someone coming along later that the 4 here correspond with the 4 on line 438.  In a future PR it might be worth figuring out how to make the relationship more obvious.